Automattic/wp-calypso
# Odyssey Stats: Link Store Stats to Woo Analytics

P2: pejTkB-1ae-p2

## Proposed Changes

* Jetpack Stats: Link Store (Shop) Tab to Woo Analytics Overview for Jetpack Stats within WP-Admin

## Testing Instructions

* Use you local dev site for testing
* Install Jetpack + WooCommerce, and connect the site
* Open `/wp-admin/admin.php?page=stats#!/stats/subscribers/:site`
* Skip Stats purchase
* Click `Store` tab
* Ensure you are taken to `/wp-admin/admin.php?page=wc-admin&path=%2Fanalytics%2Foverview`

![image](https://github.com/Automattic/wp-calypso/assets/1425433/7e05077c-5949-48a4-9dda-d75027358a4e)


## Pre-merge Checklist
